website/content/labs/lab6.md

```diff
@@ -2,192 +2,220 @@
 
-# Task 4: GitHub CI
 
-[10 points]
 
-Most version control systems include or integrate well with CI systems, and GitHub is no exception. It actually works with a number of different [CI systems](https://github.com/marketplace/category/continuous-integration), although a popular option is [Travis CI](https://travis-ci.org/)
+# Task 3: More build configurations
 
-In this final task, you will take what you've learned about CI on GitLab, and see how it translates to a different context. So, this task is more about how to familiarize yourself with a new system (and knowing how to read documentation), than about CI specifics. In fact, all we want you to do in this task is the following:
+In the previous tasks, we've given you the exact configurations you had to use (and asked for the build URLs to make sure you actually worked through the instructions in the lab). In this task, it's time for you to figure out how to write certain build configurations on your own by reading the Travis CI documentation. More specifically, you will want to consult the [Travis CI User Documentation](https://docs.travis-ci.com/) and the [Travis CI Build Config Reference](https://config.travis-ci.com/). Travis CI also provides a [Config Explorer](https://config.travis-ci.com/explore) that can allow you to validate that your `.travis.yml` is correct before you commit/push it and trigger a build.
 
-* Create a repository under your personal GitHub account called `cs220-lab6`.
-* Add the `hello/` files to the repository.
-* Setup Travis CI on your repository to build the `hello` binary and run the `hello` program.
+* [10 points] In Task 2, we saw that using the `compiler` option resulted in the `clang` build still using GCC. There is something you will need to fix in your `Makefile` Hint: The Travis CI documentation includes language-specific documentation that explains what certain options, like the `compiler` option, do. You may want to look at that specifically.
 
-Note: Doing this will involve writing a `.travis.yml` file similar to the `.gitlab-ci.yml` you wrote in the previous tasks. However, take into account that Travis CI and Gitlab CI are two completely different systems and, while they may involve similar concepts (builds, stages, etc.), you cannot simply use a `.gitlab-ci.yml` file in Travis CI. Hint: We use Travis CI in the chiventure repository, and we encourage you to look at the `.travis.yml` file in the root of the repository. You can also see the result of the latest CI build (as well as all past builds) here: https://travis-ci.org/uchicago-cs/chiventure
+  Create a build with two jobs (one with GCC and one with clang), and make sure that the clang one is actually using clang (by looking at the output of the Make commands in the "Job log"). Take the URL of the build you just produced, and paste it into Gradescope (under "Task 3: GCC vs clang"). You must also explain what you did to ensure the clang job used clang.
```

Comment:
Should students remove the "include" section from the travis.yml file and go back to using compiler, etc.? If so, it might be useful to specify that here.

Reply:
Clarified that they should only be using the compiler option and that, if they end up with more than two jobs, that means they still have options present that can affect the jobs that get launched.